kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/SubscriptionServiceImpl.java

```diff
@@ -423,41 +422,29 @@ public void deactivateSubscription(Subscription subscription, DeactivationReason
     }
 
     @Override
+    @Cacheable("pack-type")
```

Comment:
I don't see the distinction between domain and non-domain.  Also what non-domain entities would we cache?  It seems like we should have a single cache per "thing"